pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-7581-add-feature-flag-export-new.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet  author="dbush"  id="add-feature-flag-export-new">
+        <insert  tableName="access_role">
+            <column  name="name"  value="feature_flag_export_new"/>
```

Comment:
should we still add the flag to the permissions even if we are not rolling out the change per feature flag?

Reply:
good catch